content/en/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -58,36 +73,94 @@ endpoints:
+By default, Endpoint Slices managed by the Endpoint Slice controller will have no
 more than 100 endpoints each. Below this scale, Endpoint Slices should map 1:1
 with Endpoints and Services and have similar performance.
 
 Endpoint Slices can act as the source of truth for kube-proxy when it comes to
 how to route internal traffic. When enabled, they should provide a performance
 improvement for services with large numbers of endpoints.
 
-## Address Types
+### Address Types
 
-EndpointSlices support three address types:
+Endpoint Slices support three address types:
 
 * IPv4
 * IPv6
 * FQDN (Fully Qualified Domain Name)
 
-## Motivation
-
-The Endpoints API has provided a simple and straightforward way of
-tracking network endpoints in Kubernetes. Unfortunately as Kubernetes clusters
-and Services have gotten larger, limitations of that API became more visible.
-Most notably, those included challenges with scaling to larger numbers of
-network endpoints.
-
-Since all network endpoints for a Service were stored in a single Endpoints
-resource, those resources could get quite large. That affected the performance
-of Kubernetes components (notably the master control plane) and resulted in
-significant amounts of network traffic and processing when Endpoints changed.
-Endpoint Slices help you mitigate those issues as well as provide an extensible
-platform for additional features such as topological routing.
+### Topology
+
+With Endpoint Slices, each endpoint can contain relevant topology information.
+Endpoint Slices managed by the Endpoint Slice controller will contain the
+following information when it's available:
+
+* `kubernetes.io/hostname` - The name of the Node this endpoint is on.
+* `topology.kubernetes.io/zone` - The zone this endpoint is in.
+* `topology.kubernetes.io/region` - The region this endpoint is in.
+
+### Management
+
+By default, Endpoint Slices will be created and managed by the Endpoint Slice
+controller, but they can be useful beyond that. Endpoint Slices may be useful
+for a variety of additional use cases such as service mesh implementations. To
+ensure that multiple entities can manage Endpoint Slices without interfering
+with each other, a `endpointslice.kubernetes.io/managed-by` label is used to
+indicate the entity managing an Endpoint Slice. The Endpoint Slice controller
+sets `endpointslice-controller.k8s.io` as the value for this label on all
+Endpoint Slices it manages. Other entities managing Endpoint Slices should also
+set a unique value for this label.
+
+## Endpoint Slice Controller
+
+The Endpoint Slice controller watches Services and Pods to ensure corresponding
+Endpoint Slices are up to date. The controller will manage Endpoint Slices for 
+every Service with a Selector specified. These will represent the IPs of Pods
+matching the Service Selector. 
+
+### Size of Endpoint Slices
+
+By default, Endpoint Slices will be limited to a size of 100 endpoints each. This
```

Comment:
If you write this paragraph in the active voice, it kind-of forces you to tell the reader what imposes the limit (which can be as broad as “Kubernetes”). I recommend active voice here.

Reply:
Updated, thanks!